src/lib/viewers/doc/DocControls.tsx

```diff
@@ -65,9 +68,11 @@ export default function DocControls({
                     onAnnotationModeEscape={onAnnotationModeEscape}
                 />
             </ControlsBar>
-            <ControlsBar>
-                <ColorPickerControl annotationMode={annotationMode} onAnnotationColorClick={onAnnotationColorClick} />
-            </ControlsBar>
+            {annotationMode === AnnotationMode.DRAWING && (
```

Comment:
Should this also be behind a `hasDrawing` check for consistency with the main controls?

Reply:
Yes that sounds good.